backend/posts/models.py

```diff
@@ -0,0 +1,32 @@
+import uuid
+from django.db import models
+from authors.models import Author
+
+class Post(models.Model):
+    CONTENT_TYPES = [
+        ("text/markdown", "text/markdown"),
+        ("text/plain", "text/plain"),
+        ("application/base64","application/base64"),
+        ("image/png;base64","image/png;base64"),
+        ("image/jpeg;base64","image/jpeg;base64")
+    ]
+    VISIBILITY_TYPES = [
+        ("PUBLIC", "PUBLIC"),
+        ("FRIEND", "FRIEND")
+    ]
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    #do we want the url and host as a field?
+    source = models.URLField(max_length=200)
+    origin = models.URLField(max_length=200)
+    description = models.CharField(max_length=200)
+    contentType = models.CharField(max_length=200, choices = CONTENT_TYPES, default= "text/plain")
+    author = models.ForeignKey(Author, on_delete = models.SET_NULL, null=True)
+    published = models.DateTimeField(auto_now_add=True, blank=True)
+    visibility = models.CharField(max_length=200, choices = VISIBILITY_TYPES, default= "PUBLIC")
+    unlisted = models.BooleanField(default=False)
+
+#currently not being saved, need to figure out how to save nested categories
```

Comment:
@jonny21099 I currently have another model defined for `categories`, as `categories` is a list of strings. Should we store a list of strings a different way?

Reply:
We could use another model if you think it's easier to reference, but I think if you want the simple way of doing this either a JSONField or a TextField that we populate ourselves would work for this

Reply:
I've tried using the a List.Serializer like so in pasts/serializers.py like so:
`    categories = serializers.ListSerializer(child=serializers.CharField(), required=False)`
but the resulting payload looks like this
`"author": "00cdb8b3-35fa-4a91-8c06-19f29bcf1401",
        "categories": [
            "[",
            "'",
            "w",
            "e",
            "b",
            "'",
            ",",
            " ",
            "'",
            "t",
            "u",
            "t",
            "o",
            "r",
            "i",
            "a",
            "l",
            "'",
            "]"
        ],
        "count": "0",`
May need some assistance on this @jonny21099